lib/Core/Searcher.h

```diff
@@ -234,14 +234,18 @@ namespace klee {
     typedef std::tuple<ExecutionState*, size_t, size_t> priority_tuple;
```

Comment:
You don't need a tuple because you're not using std::less anymore. Use a struct please, because `get<0>` means nothing.

Reply:
Uh, yeah, I'm still using std::less (see priority_less below).

And sure.

Reply:
I saw that priority_less inherits from std::less, but then you also override operator(), so the inheritance isn't required i don't think.